DEVELOP.rst

```diff
@@ -1,68 +1,72 @@
-===============
-Developer Guide
-===============
-
-
-Prerequisites
-=============
-
-You will need Python 3.
+###############
+Developer guide
+###############
 
 
+*****
 Setup
-=====
+*****
 
-Installing node locally::
+Install Node.js locally::
 
     python3 -m venv .venv
     source .venv/bin/activate
     pip install -U pip
     pip install nodeenv
-    nodeenv --python-virtualenv --node=14.15.4
+    nodeenv --python-virtualenv --node=15.10.0
 
 Install the package dependencies::
 
     npm install
 
 
-Building the App
-================
+*****
+Tests
+*****
 
-Build the app like so::
+To run the tests, use the following commands::
 
-    npm run build
+    npm test
 
-To run the Admin UI as a standalone app, open the ``build/index.html`` file in
-your browser.
 
 
-Running the App for Development
-===============================
+***********************
+Development environment
+***********************
 
-Start the development server on port ``9000``::
+Quickstart
+==========
 
-    npm start
+Start CrateDB::
 
-You should now be able to access the app at: http://localhost:9000/
+    npm run cratedb
+
+Start the development server on port ``9000``::
 
+    npm run develop
 
-Running CrateDB
-===============
+After the application has been compiled and the Webpack development server has
+started, the process should automatically open a browser tab.
 
-The quickest way to spin up an instance is probably to use Docker::
 
-    docker run -it --rm --publish 4200:4200 crate/crate:nightly -Chttp.cors.enabled=true -Chttp.cors.allow-origin=*
+Description
+===========
 
+You should now be able to access the app at: http://localhost:9000/
 
 Connecting to CrateDB
-=====================
+---------------------
 
 You will need to enable *Cross-Origin Resource Sharing* in your `CrateDB
 configuration`_ to test the Admin UI against it::
 
     http.cors.enabled: true
     http.cors.allow-origin: "*"
 
+A quick way to spin up an instance of this is to use Docker::
+
+    docker run -it --rm --publish 4200:4200 crate/crate:nightly -Chttp.cors.enabled=true -Chttp.cors.allow-origin=*
+
```

Comment:
This section confuses me. The above sections tells me that I can/should start crate by `npm run cratedb`.
So its already started including the cors config.
I guess this section should server as an alternative to `npm run cratedb`, so we should definitly highlight it as as alternative ;)

Reply:
Or maybe delete it completely and just talk about the `base_uri` in order to connect to different instances

Reply:
> I guess this section should server as an alternative to `npm run cratedb`.

That was its original intention, right.

> Or maybe delete it completely and just talk about the `base_uri` in order to connect to different instances.

You are right. I amended this again on behalf of 87c99096c.